### PR TITLE
Optimize part of the code, correct misspellings

### PR DESCRIPTION
**Pull request by flyingpig233-cpu**
I found many typos in the comments

### CODE REVIEW

File: src/memman.c [hunk 1]
```diff
@@ -1,188 +1,208 @@
...
-    // Exit scope
-    while (self->scope_depth) {
-        fprintf(stderr, "libcmemman: warning: scope %d never exits until destroying the pool.", self->scope_depth);
-        mp_exit_scope(self);
-    }
-
-    // Free the pool and the object itself.
-    free(self->ptr_pool);
-    free(self);
+	// Exit scope
```
Comment (onion108) [inline src/memman.c:21]: Please repect my indent style, bro

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ And after that, we enter a new scope:
  |     \
  \
 ```
-And we allocated some blockes of memory and stored it into `b`, `c`, `d`.
+And we allocated some blocks of memory and stored it into `b`, `c`, `d`.
```
Comment (onion108) [inline README.md:92]: well, typo

**Review by onion108 — COMMENTED** [review]
Use the original indent.

File: src/memman.c (hunk 1 above)
Comment (flyingpig233-cpu) [reply src/memman.c:21]: clang-format shit

**Comment by flyingpig233-cpu** [comment]
I see.

**Comment by flyingpig233-cpu** [comment]
it's duty for you

**Comment by onion108** [comment]
I will fix the typo by myself (use your forked repo as a reference), but I'm afraid I won't merge it 'cause of the indent style

**Pull request closed without merging**